### PR TITLE
[Ra2 ch3, ch4] language alterations

### PR DESCRIPTION
**Pull request by mpgreaves**

### CODE REVIEW

**Review by steelescotr — APPROVED** [review]

**Review by petorre — APPROVED** [review]

**Review by rgstori — APPROVED** [review]

File: doc/ref_arch/kubernetes/chapters/chapter03.rst [hunk 1]
```diff
@@ -408,27 +413,30 @@ Standard <https://github.com/k8snetworkplumbingwg/multi-net-spec/tree/master/v1.
...
 
 :numref:`Kubernetes Networking Architecture` below shows the main building blocks of a Kubernetes networking solution:
 
--  **Kubernetes Control Plane**: this is the core of a Kubernetes Cluster - the
-   apiserver, etcd cluster, kube-scheduler and the various controller-managers. The
-   control plane (in particular the apiserver) provide a centralised point by which
-   the networking solution is managed using a centralised management API.
+-  **Kubernetes Control Plane**: this is the core of a Kubernetes Cluster: the apiserver, the etcd cluster,
+   the kube-scheduler, and the various controller-managers. The control plane (in particular the apiserver)
+   provides a centralized point by which the networking solution is managed using a centralized management API.
```
Comment (rgstori) [inline doc/ref_arch/kubernetes/chapters/chapter03.rst:431]: british spelling
```suggestion
   provides a centralised point by which the networking solution is managed using a centralised management API.
```

**Review by rgstori — APPROVED** [review]
approved, just a couple spelling fixes

File: doc/ref_arch/kubernetes/chapters/chapter07.rst [hunk 2]
```diff
@@ -148,7 +148,7 @@ HW topology aware huge pages
 **Baseline project:** *Kubernetes*
 
 **Gap description:** Memory Manager was added in v1.21 as alpha feature. More in
-:ref:`chapters/chapter03:memory and huge pages resources management`.
+:ref:`chapters/chapter03:management of memory and huge pages resources`.
```
Comment (rgstori) [inline doc/ref_arch/kubernetes/chapters/chapter07.rst:151]: trailing whitespace here

**Review by rgstori — CHANGES_REQUESTED** [review]
trailing whitespace

**Comment by CsatariGergely** [comment]
> trailing whitespace

@rgstori this is fixed now and ready to be merged.

**Review by rgstori — APPROVED** [review]

**Pull request merged**